content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+---
+title: Multicluster Installation
+description: Install an Istio mesh across multiple Kubernetes clusters.
+weight: 30
+aliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+keywords: [kubernetes,multicluster]
+test: yes
```

Comment:
Although you do have test: set to yes which generates the snips.sh, and there are not test scripts created. The yes does show a checkmark on the page that this page is tested, when in fact there are no tests.

That said, I noticed that the scrips.sh did get lint tested to highlight problems in the snippets on the page.

Reply:
@frankbu added `yes` ... I assumed that was the goal, but not sure. I'm fine setting it back to `no`. I'd like to do the tests with a follow-on PR since it will be huge and complex.

Reply:
OK. I'll let @frankbu respond if it should go back to no now that the scripts are tested and being there are no tests at the moment. As noted above, I can see a reason to have the scripts.sh created and linted when updating pages with snippets. But it also leaves a false impression that a page is tested.

Reply:
As long as the test is coming in a followup PR, before the 1.8 release (which is our policy for new docs), I think it's OK if it's set to yes in preliminary.istio.io, so that (as @ericvn pointed out) we can at least be sure the commands have no lint errors.